docs/sources/upgrading/_index.md

```diff
@@ -38,6 +38,14 @@ The output is incredibly verbose as it shows the entire internal config struct u
 We now evenly spread queriers across the available kubernetes nodes, but allowing more than one querier to be scheduled into the same node.
 If you want to run at most a single querier per node, set `$._config.querier.use_topology_spread` to false.
 
+#### Default value for `server.http-listen-port` changed
```

Comment:
I think this should be under the Loki section of 2.6.0? looks like it got out of place?

Reply:
oh interesting, this PR moves it, is this because it's not in the 2.6.0 release?

Reply:
Christian added it to the bottom of the section